cypress/integration/accessibility_Popover_spec.js

```diff
@@ -5,6 +5,18 @@ describe('Popover Accessibility check', () => {
   });
 
   it('Tests accessibility on the Popover page', () => {
+    cy.configureAxe({
+      rules: [
+        {
+          id: 'color-contrast', //  Element has insufficient color contrast of 1.29 (foreground color: #111111, background color: #2a2734, font size: 12.0pt (16px), font weight: bold). Expected contrast ratio of 4.5:1
+          enabled: false,
+        },
+        {
+          id: 'region', //  Some page content is not contained by landmarks: <div class="Dl7 rTt mi- ho- ygj _O1 XbT DHH kFh" style="max-width: 180px;"><div class="Zr3 hUC zI7 iyn Hsu"><div class="tBJ dyH iFc yTZ erh tg7 IZT swG">Filter your board to see your favorite Pins, and more</div></div></div>
```

Comment:
@ayeshakmaz If you could check this one, I'd appreciate it.

Reply:
(resolved in Slack)